packages/lesswrong/components/common/HeaderEventSubtitle.tsx

```diff
@@ -31,6 +34,18 @@ type CurrentEvent = {
 
 const useCurrentEvent = (): CurrentEvent | null => {
   const spotlight = useCurrentFrontpageSpotlight();
+  
+  // special case for EA Forum Giving Season 2023
+  const now = moment()
+  const isGivingSeason = isEAForum && moment(timelineSpec.start).isBefore(now) && moment(timelineSpec.end).isAfter(now)
+  if (!spotlight && isGivingSeason) {
+    return {
+      title: 'Giving season 2023',
+      link: '/giving-portal',
+      background: makeBackground('#862115', '#E7714E')
+    }
+  }
```

Comment:
The header event subtitle is otherwise tied to spotlights, and this seemed like the simplest way to make the giving season one work.